test/internet/test-dns-idna2008.js

```diff
@@ -41,16 +41,16 @@ dns.promises.lookup(fixture.hostname, { family: fixture.family })
   .then(({ address }) => {
     assert.strictEqual(address, fixture.expectedAddress);
   }, (err) => {
-    if (err && err.errno === 'ESERVFAIL') {
-      assert.ok(err.message.includes('queryA ESERVFAIL straße.de'));
+    if (err && err.code === 'ENOTFOUND') {
+      assert.ok(err.message.includes('queryA ENOTFOUND ENOTFOUND.de'));
```

Comment:
I think this should be

```suggestion
      assert.ok(err.message.includes('queryA ENOTFOUND straße.de'));
```

no?

Reply:
Yes!